webpack.config.js

```diff
@@ -28,6 +28,7 @@ const basePlugins = [
       },
     },
   }),
+  new webpack.optimize.ModuleConcatenationPlugin(),
```

Comment:
This should improve JS bundle size when we switch to using more ES modules -- but we're not there yet.